.github/CODEOWNERS

```diff
@@ -27,7 +27,6 @@
 /src/workspaces/ @DataBiosphere/broadworkspaces-terra-ui
 /integration-tests/tests/billing-projects.js @DataBiosphere/broadworkspaces-terra-ui
 /integration-tests/tests/delete-orphaned-workspaces.js @DataBiosphere/broadworkspaces-terra-ui
-/integration-tests/tests/workspace-dashboard.js @DataBiosphere/broadworkspaces-terra-ui
```

Comment:
This test was deleted in an earlier PR, and I forgot to remove the CODEOWNERS entry.